src/Common/ThreadFuzzer.h

```diff
@@ -54,13 +54,19 @@ class ThreadFuzzer
 
     bool isEffective() const;
 
+    static void stopFuzzing();
+    static void startFuzzing();
+    static bool isStopped();
+
 private:
     uint64_t cpu_time_period_us = 0;
     double yield_probability = 0;
     double migrate_probability = 0;
     double sleep_probability = 0;
     double sleep_time_us = 0;
 
+    inline static std::atomic<bool> stop_fuzzing{false};
```

Comment:
Negative meaning of this variable is unnatural.
Maybe name it `active` and set to true?

Reply:
But it can be confused with `isEffective`

Reply:
Then may be `started`?